pysaliency/external_datasets/nusef.py

```diff
@@ -117,20 +128,25 @@ def get_NUSEF_public(location=None):
                          eye_scn_dist,
                          no_of_flags,
                          fix_loss,
-                         interfix_loss) = line.split()
+                         interfix_loss) = lines[i].split()
                         x.append(float(hor_pos) * scale_x)
                         y.append(float(ver_pos) * scale_y)
-                        t.append(float(start_time.split(':')[-1]))
+                        if i == 0:
+                            initial_start_time = datetime.strptime(str(start_time), date_format)
+                        current_start_time = datetime.strptime(str(start_time), date_format) 
```

Comment:
maybe more concise and easier to read:
```suggestion
                        current_start_time = datetime.strptime(str(start_time), date_format)
                        if i == 0:
                            initial_start_time = current_start_time
```